ush/predef_grid_params.yaml

```diff
@@ -308,7 +308,7 @@
   ESGgrid_NY: 200
   ESGgrid_PAZI: 0.0
   ESGgrid_WIDE_HALO_WIDTH: 6
-  DT_ATMOS: 40
+  DT_ATMOS: 36
   LAYOUT_X: 5
```

Comment:
What was the justification for decreasing the time step here and above?

Reply:
Got it!  It's probably a good idea to keep them consistent.  We also may want to just adopt the final RRFS-based time step once all code has been transitioned to the SRW App.